packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -782,6 +771,12 @@ export class BaseResolversVisitor<
         prev[typeName] = applyWrapper(this.config.defaultMapper.type);
       } else if (isScalar) {
         prev[typeName] = applyWrapper(this._getScalar(typeName));
+      } else if (isInterfaceType(schemaType)) {
+        this._hasReferencedResolversInterfaceTypes = true;
+        const type = this.convertName('ResolversInterfaceTypes');
+        const generic = this.convertName(currentType);
+        prev[typeName] = applyWrapper(`${type}<${generic}>['${typeName}']`);
+        return prev;
```

Comment:
This Interface block must be below `hasDefaultMapper && !hasPlaceholder(this.config.defaultMapper.type)` if block. 

Otherwise, the default mapper case with no placeholder e.g. `config.defaultMapper: any` will incorrectly generate ResolversInterfaceTypes